docs/topics/migrations.txt

```diff
@@ -305,6 +305,27 @@ migrations are treated no differently from any other migration.
 
 .. _migration-history-consistency:
 
+Non-atomic migrations
+---------------------
```

Comment:
I think this should be a main topic, maybe below the `Workflow`, e.g.
```suggestion
Transactions
============
```

Reply:
Agreed!